kong/plugins/http-log/handler.lua

```diff
@@ -189,12 +190,27 @@ function HttpLogHandler:log(conf)
 
   local queue_conf = Queue.get_plugin_params("http-log", conf, make_queue_name(conf))
   kong.log.debug("Queue name automatically configured based on configuration parameters to: ", queue_conf.name)
+  local entry = cjson.encode(kong.log.serialize())
+
+  if queue_conf.max_batch_size == 1 then
```

Comment:
If this is a hidden feature. We should update the official doc https://docs.konghq.com/gateway/latest/kong-plugins/queue.

Otherwise, add `no_queue` looks good.

Reply:
We're waiting for the internal discussion.